dashboards/clusterbytenant.json

```diff
@@ -1860,7 +1825,7 @@
         "current": {
           "selected": false,
           "text": "cluster-monitoring",
-          "value": "cluster-monitoring"
+          "value": "PE8D8DB4BEE4E4B22"
```

Comment:
Grafana generated change, i guess this is a related to grafana version

Reply:
It is better to manually edit the query instead of re-exporting the dashboard. Otherwise, it contains too many changes, completely unrelated to the change description.